packages/EasyBatch/src/BatchObjectManager.php

```diff
@@ -73,10 +72,6 @@ public function approve(BatchObjectInterface $batchObject): BatchObjectInterface
                 $this->approve($this->batchItemRepository->findOrFail($batchObject->getParentBatchItemId()));
             }
 
-            // Approving a batch is possible only when all its batchItems succeeded and approval was required
```

Comment:
We have dispatch event next in a `\EonX\EasyBatch\Processors\BatchProcessor::dispatchBatchEvent`

Reply:
@alexndlm this event dispatch is to handle the case when we are approving the batch itself which in this case we don't go through the batch processor, or maybe I don't understand the issue.
Could you please explain how to reproduce the bug? Thanks

Reply:
I have rollbacked this. I will research this issue in addition.